src/parser/paladin/protection/modules/spells/WordOfGlory.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent, HealEvent, ApplyBuffEvent, RemoveBuffEvent } from 'parser/core/Events';
+import { When, ThresholdStyle } from 'parser/core/ParseResults';
+import { Options } from 'parser/core/Module';
+import HIT_TYPES from 'game/HIT_TYPES';
+
+const OVERHEAL_THRESHOLD = 0.75;
+
+export default class WordOfGlory extends Analyzer {
+  _sl = {
```

Comment:
In general I find it cleaner to make all these separate props. Mutating objects leads to bugs quicker, and objects are inherently more complex.